README.md

```diff
@@ -27,22 +31,23 @@
 - Configurable reverse geocoding server
 - [A lot more in commit history](https://github.com/ffrgb/meshviewer/commits/develop)
 
-# Demo (embedded):
+## Demo:
 
-https://regensburg.freifunk.net/netz/karte/
+https://regensburg.freifunk.net/netz/karte/ embedded and<br>
+https://regensburg.freifunk.net/meshviewer/ standalone
```

Comment:
https://map.bremen.freifunk.net/ (with a patch for a different `nodes.json` generated by [https://github.com/FreifunkBremen/respond-collector](respond-collector )